src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -184,6 +191,21 @@ protected SourceSection findSourceLocation(RubyContext context, Object value) {
         }
     }
 
+    @Override
+    protected List<OptionDescriptor> describeOptions() {
+        final List<OptionDescriptor> options = new ArrayList<>();
+
+        for (OptionDescription<?> option : OptionsCatalog.allDescriptions()) {
+            // TODO CS 19-Jun-17 default value of null isn't supported
```

Comment:
Maybe we should just use an empty String for those options with a `null` default?
From a quick look it would not be conflicting for the options using `null`, as the empty String would be an invalid value for these options anyway.